messages/variables/messages.go

```diff
@@ -44,13 +42,18 @@ var (
 	UpdateHelpFlag         = "Displays more information about the update subcommand"
 
 	// [ create ]
-	CreateUsage            = "create [flags]"
 	CreateShortDescription = "Creates a new environment variable or secret"
 	CreateLongDescription  = "Creates a new environment variable or secret to be used inside Edge Functions"
 	CreateFlagKey          = "Informs the variable's key"
 	CreateFlagValue        = "Informs the variable's value"
 	CreateFlagSecret       = "Indicates whether the value is meant to be confidential."
-	CreateFlagIn           = "Path to a JSON file containing the attributes of the variable that will be created; you can use - for reading from stdin"
+	CreateFlagFileJSON     = "Path to a JSON file containing the attributes of the variable that will be created; you can use - for reading from stdin"
 	CreateOutputSuccess    = "Created variable with UUID %s\n"
 	CreateHelpFlag         = "Displays more information about the create subcommand"
+
+	// [ ask ]
+	AskKey        = "Enter your key:"
+	AskValue      = "Enter your value:"
+	AskSecret     = "Enter your secret:"
+	AskVariableID = "Enter your variable ID:"
```

Comment:
We could use "Enter the Variable's key/value/secret/ ID" instead of `your`

Reply:
whatever